### PR TITLE
Publishing shared animation

### PR DESCRIPTION
**Pull request by Andrew-K-Lam**
## Description
Publishing shared animation

## Checklist before submitting pull request

- [x] New code is unit tested
- Commits follow our [Developer Guide](https://tds.telus.com/contributing/developer-guide.html#make-a-commit)
- [x] For code changes, run `npm run prepr` locally
  - make sure visual and accessibility tests pass

### CODE REVIEW

File: scripts/build.js [hunk 1]
```diff
@@ -27,16 +27,15 @@ getPackageNames(packageNames => {
       'exec',
       '--scope',
       scopeGlob,
-      '--ignore',
-      '@tds/shared-*',
+      '--no-private',
```
Comment (theetrain) [inline scripts/build.js]: Will this no longer create version bumps for private packages?

Comment (jraff) [reply scripts/build.js]: https://www.npmjs.com/package/@lerna/filter-options#--no-private

`@tds/shared-*` components used to be private packages that were used in tds-core.

If my understanding is correct, now that we are publishing components with the `@tds/shared-*` prefix we can no longer ignore them, instead we can use [`--no-private`](https://www.npmjs.com/package/@lerna/filter-options#--no-private) to exclude all private packages.

File: packages/Select/__tests__/Select.spec.jsx [hunk 2]
```diff
@@ -6,9 +6,9 @@ import Text from '@tds/core-text'
 import Paragraph from '@tds/core-paragraph'
 import InputFeedback from '@tds/core-input-feedback'
 import DecorativeIcon from '@tds/core-decorative-icon'
+import { Fade } from '@tds/shared-animation'
```
Comment (theetrain) [inline packages/Select/__tests__/Select.spec.jsx]: Was this reflected in its package.json?

File: packages/TextArea/__tests__/TextArea.spec.jsx [hunk 3]
```diff
@@ -5,9 +5,9 @@ import StandaloneIcon from '@tds/core-standalone-icon'
 import Text from '@tds/core-text'
 import Paragraph from '@tds/core-paragraph'
 import InputFeedback from '@tds/core-input-feedback'
+import { Fade } from '@tds/shared-animation'
```
Comment (theetrain) [inline packages/TextArea/__tests__/TextArea.spec.jsx]: Was this reflected in its package.json?

File: scripts/build.js (hunk 1 above)
Comment (theetrain) [reply scripts/build.js]: I see, so this only excludes private packages from **build**, and none of them have a rollup config, so they wouldn't build anyway.

File: packages/Select/__tests__/Select.spec.jsx (hunk 2 above)
Comment (Andrew-K-Lam) [reply packages/Select/__tests__/Select.spec.jsx]: Yes: https://github.com/telus/tds-core/blob/2ae6bdb60a46f22854c5ee965379bf0597bdb640/packages/Select/package.json#L38

File: packages/TextArea/__tests__/TextArea.spec.jsx (hunk 3 above)
Comment (Andrew-K-Lam) [reply packages/TextArea/__tests__/TextArea.spec.jsx]: Yes https://github.com/telus/tds-core/blob/4e1224d72a56a3b0a140b62c27fbcd82f0612add/packages/TextArea/package.json#L37

File: packages/Select/__tests__/Select.spec.jsx (hunk 2 above)
Comment (theetrain) [reply packages/Select/__tests__/Select.spec.jsx]: From this commit hash, I see that it is still in `devDependencies`

https://github.com/telus/tds-core/blob/2ae6bdb60a46f22854c5ee965379bf0597bdb640/packages/Select/package.json#L36-L38

Comment (theetrain) [reply packages/Select/__tests__/Select.spec.jsx]: My mistake, this should remain in `devDependencies` since it's only being used in tests.

**Pull request merged**